docs/sources/guides/whats-new-in-v6-6.md

```diff
@@ -101,4 +115,57 @@ Paste a native emoji in the unit picker and pick it as a custom unit:
 
 ## Cookie management modifications
 
-In order to align with a [change in Chrome 80](https://www.chromestatus.com/feature/5088147346030592), a breaking change has been introduced. The `[security]` setting `cookie_samesite` configured to `none` now renders cookies with `SameSite=None` attribute contrary to the previous behavior where no `SameSite` attribute was added to cookies. To get back the old behavior, you must set `cookie_samesite` to `disabled`.
+In order to align with a [change in Chrome 80](https://www.chromestatus.com/feature/5088147346030592), a breaking change has been introduced to Grafana's [`cookie_samesite` setting]({{< relref "../installation/configuration.md#cookie-samesite" >}}). Grafana now properly renders cookies with the `SameSite=None` attribute when this setting is `none`. The previous behavior of `none` was to omit the `SameSite` attribute from cookies. Grafana will use the previous behavior when `cookie_samesite` is set to `disabled`.
+
+## Explore/Logs Panel: Log message line wrapping options
+
+We are introducing wrap-lines option for logs, as for some of our users it's more efficient to see one line per log message.The wrapped-line option is set as a default, unwrapped setting results in horizontal scrolling.
+
+{{< docs-imagebox img="/img/docs/v66/explore_wrap_lines.gif" max-width="800px" caption="Log message line wrapping" >}}
+
+## Explore/Logs Panel: Column with unique log labels
+
+After feedback from our community, we have decided to reintroduce a labels column. However, for better readability and usefulness, we have transformed it into Unique labels column and include only non-common labels, as all common labels are displayed above.
+
+{{< docs-imagebox img="/img/docs/v66/explore_labels_column.png" max-width="800px" caption="Unique log labels column" >}}
+
+## Explore: Context tooltip
+
+Isolating a series from a big set of lines in a graph is important for drill-downs. That's why we have implemented context tooltip in Explore that allows interaction to copy data and labels from it to further refine the query.
+
+{{< docs-imagebox img="/img/docs/v66/explore_context_tooltip.png" max-width="800px" caption="Explore context tooltip" >}}
+
+## Enterprise: White labeling
+
+This release adds new white labeling options to the grafana.ini file (can also be set via ENV variables).
+
+```ini
+[white_labeling]
```

Comment:
I think we should use the ini from sample instead of defaults.

```ini
# Enterprise only
[white_labeling]
# Set to complete url to override login logo
;login_logo =

# Set to complete css background expression to override login background
# example: login_background = url(http://www.bhmpics.com/wallpapers/starfield-1920x1080.jpg)
;login_background =

# Set to complete url to override menu logo
;menu_logo =

# Set to complete url to override fav icon (icon shown in browser tab)
;fav_icon =

# Set to complete url to override apple/ios icon
;apple_touch_icon =

# Below is an example for how to replace the default footer & help links with 2 custom links
;footer_links = support guides
;footer_links_support_text = Support
;footer_links_support_url = http://your.support.site
;footer_links_guides_text = Guides
;footer_links_guides_url = http://your.guides.site
```

Reply:
changed it to this but uncommented with example values